<launch>
	<group ns="buoy">
		<node name="plot_streaming_data_node" pkg="acoustic" type="plot_streaming_data_V7.py" output="screen" clear_params="true">
			<rosparam command="load" file="$(find acoustic)/config/plot_streaming_data.yaml"/>
		</node>
	</group>
</launch>
